modules/customize-certificates-api-add-named.adoc

```diff
@@ -18,6 +18,17 @@ client's URL.
 reach the API server.
 * The certificate must have the `subjectAltName` extension for the URL.
 
+If a certificate chain is used, all certificates must be manually concatenated into a single named certificate file. This can be achieved with the following example: cat first_cert.pem second_cert.pem third_cert.pem >combined_cert.pem 
```

Comment:
Can you put backticks around "cat first_cert.pem second_cert.pem third_cert.pem >combined_cert.pem" and add a period at the end?

Reply:
I got really sidetracked and forgot about this until it came up today. Appropriate edits made and I believe I have now squash properly (pick, squash, squash), it's been a while thought might have hosed it, apologies.

Reply:
@pbmoses The edits look great, but unfortunately I still see 6 commits (you can see them in the "Commits" tab). If you don't want to mess with trying to squash anymore, just let me know and I can take care of squashing and pushing your update. Thanks!